source/how-we-work/version-control/pull-requests.html.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Working with Pull Requests
+weight: 1
+---
+
+# Working with Pull Requests (PRs)
+
+## Making changes to PRs from external contributors
+
+Occasionally, we need to help a contributor by making changes to their Pull Request before we can merge it. For example, we might offer to rebase their branch against main to save them time, or if they are not comfortable using git rebase.
+
+You should let the contributor know that you are going to make changes to their branch, especially if it involves rewriting history – where possible, ask their permission first. 
+
+Contributors do not have write access to our repo, and so when they raise a PR it comes from [their own fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks) of the repository. This means that the process for making changes is slightly different, as any changes need to be made to their fork.
+
+This requires the user to have [allowed maintainers to make changes to their branch](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/#enabling-repository-maintainer-permissions-on-existing-pull-requests). This is usually enabled by default, but if you have issues with permissions you may need to check that the user has this option enabled.
+
+The easiest way to make a change to a contributor's PR is to use the [GitHub CLI](https://cli.github.com/). Once installed, you can run `gh pr checkout 1234`, replacing `1234` with the ID of the PR you want to work on.
+
+The `gh pr checkout` command configures git so that changes are pushed to the contributor's fork when you run `git push` from the checked out branch. You can see how this works by running `git config --get-regexp branch.<BRANCH-NAME>`.
+
+Note that:
+
+- you should not specify any additional arguments when running `git push`
+- pushing the changes using a Git GUI (for example GitHub Desktop) may not work
+- pushing with `--force-with-lease` may not work (but `--force` should)
+
+## Reviewing PRs
+
+Consider who needs to review the Pull Request. If a PR includes code, design and guidance changes it may be appropriate for it to be reviewed by a developer, designer and a content designer. As a courtesy, you should consider at least giving those involved an opportunity to review it if they want to.
+
+### Reviewing a PR from a team mate
+
+PRs need to be reviewed and approved by at least 1 other team member. If you paired on a piece of work with another person, they can review and approve it.
+
+### Reviewing a PR from an external contributor
+
+PRs raised by external contributors need to be reviewed by 2 team members.
+
+When reviewing PRs from external contributors:
+
+- review the code using the GitHub interface before checking out the code and running it locally
+- be aware of [supply chain attacks](https://www.ncsc.gov.uk/collection/supply-chain-security) – check any new dependencies, and verify any changes to the lock file
```

Comment:
We've assumed knowledge of git for this documentation, but not `npm`, so we might want to link to an explanation of a lock file?